### PR TITLE
jenkins: cleanup VersionSelectorScript.groovy

### PR DESCRIPTION
**Pull request by targos**
- Remove lines that only exclude EoL versions
- Reference currently supported versions everywhere

### CODE REVIEW

File: jenkins/scripts/VersionSelectorScript.groovy [hunk 1]
```diff
@@ -20,120 +20,81 @@ def buildExclusions = [
...
-  [ /^centos7-(arm)?(64|32)-gcc6/, anyType,     gte(14) ],    // 14.x: gcc6 builds stop
-  [ /^centos7-(arm)?(64)-gcc8/,       anyType,     lt(14)  ], // 14.x: gcc8 builds start
+  [ /^centos7-(arm)?(64|32)-gcc48/,   anyType,     gte(16) ],
+  [ /^centos7-(arm)?(64|32)-gcc6/,    anyType,     gte(16) ],    // 14.x: gcc6 builds stop
   [ /^centos7-(arm)?(64)-gcc8/,       anyType,     gte(18) ], // 18.x: centos7 builds stop
-  [ /^centos7-64/,                    releaseType, lt(12)  ],
   [ /^centos7-64/,                    anyType,     gte(18) ],
-  [ /debian8-x86/,                    anyType,     gte(10) ], // 32-bit linux for <10 only
-  [ /debian8/,                        anyType,     gte(13) ],
+  [ /debian8-x86/,                    anyType,     gte(16) ], // 32-bit linux for <10 only
```
Comment (richardlau) [inline jenkins/scripts/VersionSelectorScript.groovy:27]: Do we have any debian8-x86 labelled machines (in either CI) left? If they're all gone we could remove completely.

Comment (targos) [reply jenkins/scripts/VersionSelectorScript.groovy:27]: I agree, but I prefer to do a second pass removing outdated labels.

Comment (richardlau) [reply jenkins/scripts/VersionSelectorScript.groovy:27]: SGTM

**Review by richardlau — APPROVED** [review]

**Review by UlisesGascon — APPROVED** [review]

**Comment by rvagg** [comment]
wow .. so much history in there

**Pull request merged**